LiveChat/index.js

```diff
@@ -370,6 +375,9 @@ export default class LiveChat extends Component {
 
 			customerSDK.listChats().then((data) => {
 				const { chatsSummary, totalChats } = data
+				if (!totalChats) {
+					this.removePlaceHolder();
```

Comment:
add `this.removePlaceHolder();` after `eventsToAdd` was defined, this indicates that there are messages in the flow